packages/visualizations/package.json

```diff
@@ -94,12 +95,17 @@
         "chartjs-plugin-datalabels": "^2.0.0",
         "chartjs-plugin-stacked100": "^1.2.0",
         "chroma-js": "^2.1.2",
+        "d3-geo": "^3.0.1",
+        "geojson-bbox": "^0.0.0",
+        "geojson-to-svg": "^1.1.4",
+        "geojson2svg": "^1.3.3",
```

Comment:
I think these 4 packages were used for POCs and are not used anymore?
- `geojson-bbox`
- `geojson-to-svg`
- `geojson2svg`
- `reproject-spherical-mercator`

Reply:
I removed them and I'll see if something breaks down the road 🙈